FE/todo-app/src/hooks/useModal.tsx

```diff
@@ -0,0 +1,13 @@
+import { useState } from "react";
+
+export const useModal = () => {
+  const [isOpen, setOpen] = useState(false);
+
+  const open = () => setOpen(true);
```

Comment:
tos to moc neopravil vole :D tak uplně polopaticky ... open = () => setOpen(() => true)